client/signup/steps/theme-style/index.jsx

```diff
@@ -0,0 +1,99 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import SignupActions from 'lib/signup/actions';
+import SignupThemeStyleList from './signup-theme-style-list';
+import StepWrapper from 'signup/step-wrapper';
+import Button from 'components/button';
+import { getSignupDependencyStore } from 'state/signup/dependency-store/selectors';
+
+class ThemeStyleStep extends Component {
+	static propTypes = {
+		goToNextStep: PropTypes.func.isRequired,
+		signupDependencies: PropTypes.object.isRequired,
+		stepName: PropTypes.string.isRequired,
+		translate: PropTypes.func,
+	};
+
+	pickThemeStyle = themeStyle => {
+		const { goToNextStep, signupDependencies } = this.props;
+
+		const stepThemeModInformation = {
+			active_style_pack: themeStyle.slug,
+		};
+
+		const stepSiteInformation = Object.assign( {}, signupDependencies.siteInformation, {
+			theme_mods: Object.assign(
+				{},
+				signupDependencies.siteInformation.theme_mods,
+				stepThemeModInformation
+			),
+		} );
+
+		SignupActions.submitSignupStep(
+			{
+				stepName: this.props.stepName,
+				processingMessage: this.props.translate( 'Styling your site.' ),
+			},
+			[],
+			{
+				siteInformation: Object.assign(
+					{},
+					signupDependencies.siteInformation,
+					stepSiteInformation
+				),
+			}
+		);
+
+		goToNextStep();
+	};
+
+	renderThemeStyleList() {
+		return (
+			<SignupThemeStyleList
+				theme={ this.props.signupDependencies.themeSlugWithRepo }
+				handleScreenshotClick={ this.pickThemeStyle }
+			/>
+		);
+	}
+
+	renderJetpackButton() {
+		return (
+			<Button compact href="/jetpack/connect">
+				{ this.props.translate( 'Or Install Jetpack on a Self-Hosted Site' ) }
+			</Button>
+		);
+	}
+
+	render = () => {
+		const { translate } = this.props;
+		const headerText = translate( 'Choose a style for your website.' );
+		const subHeaderText = translate( "Don't worry, you can change this later." );
```

Comment:
Hi! I've found a possible matching string that has already been translated **55** times:
`translate( 'Don't worry you can change this later.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).